content/doc/book/pipeline/syntax.adoc

```diff
@@ -697,6 +697,13 @@ pipeline {
 
 The `when` directive allows the Pipeline to determine whether the stage should
 be executed depending on the given condition.
+The `when` directive must contain at least one condition.
+If the `when` directive contains more than one condition,
```

Comment:
@daniel-beck  No, it shouldn't.  This is about about the `when` directive in general.